src/main/java/org/codinjutsu/tools/jenkins/security/JenkinsVersion.java

```diff
@@ -0,0 +1,20 @@
+package org.codinjutsu.tools.jenkins.security;
+
+/**
+ * Created by bidek on 13.01.17.
+ */
+public enum JenkinsVersion {
```

Comment:
instead of hardcoding this you can ask Jenkins server for the crumb field. 

https://github.com/pycontribs/jenkinsapi/blob/master/jenkinsapi/utils/crumb_requester.py